### ISSUE TITLE
Beehive shuffle wiggle behavior broken

### ISSUE THREAD

**Issue by rrealmuto** [TestRunnerSRL/OoT-Randomizer#1812]
As an unintended side-effect of the recent potsanity override fixes, the beehive wiggle behavior is sort of broken. It is currently being tied to the Pot/Crate Appearance Match Contents setting as follows:

If Pot/Crate Appearance is set to "Unchecked" beehives will wiggle until the item is collected.
If set to "Textures match contents" beehives will wiggle if they have a non-junk item (basically anything that would show up in a textured chest if you were to have chest textures match contents enabled). Once the item has been collected, the beehive will stop wiggling.
If set to "Off" beehives will not wiggle.

I am not strictly opposed to the behavior working this way. If kept, the setting names/descriptions should be updated to explain this behavior properly. Otherwise, the setting should be changed back to work as it previously had, and as the current description states (all beehives containing non-junk items wiggle). 

I'm open to suggestions.

**Comment by r0bd0g** [comment]
I think a separate setting for the beehive wiggle is probably appropriate. I appreciate the option to not have the beehives always spoiler whether they hold something useful, thus always forcing me to backtrack if I don't have the ability to check it right now. I don't think most people would play without the setting on, so just default it to on. Combining it with an unrelated setting (currently is it even possible to change this setting in the GUI if pots and crates are not shuffled? haven't checked) I don't think is a good idea, especially since the way it reveals the contents of the check is so different.

EDIT: I didn't realize that the 'unchecked' texture setting was also applied for beehive wiggle. I think b/c beehives are closed up and don't have an open state the way chests do, it kind of makes some sense, though I don't know if I really want a heavy-duty new drop down just for a beehive wiggle setting.

**Comment by r0bd0g** [comment]
I was thinking a lot about these settings.

I think from a gameplay perspective, small crates is a lot more like pots, and so it would make sense to be able to shuffle those either together with pots only or independently from crates. The main reason I think crates needs to be separate is because of bonko making them just horrible to play. And to have all of the texture contents settings be separate like this, that's a lot of settings. Perhaps a multiselect would be appropriate except so many of these texture settings have variants that wouldn't really work in a multiselect.

Anyway long story short is, nevermind, just combine it with pot/crates.

**Comment by cjohnson57** [comment]
Yeah, I'm okay with this behavior as well, just updating the tooltips should be fine.

**Comment by r0bd0g** [comment]
It's also somewhat weird to me that chests revealing their contents is a separate setting, but all of the other types are grouped up as one setting. No idea what to do about it.

**Comment by rrealmuto** [comment]
Closing this as we agreed to leave the behavior as is and just update the tooltip.
